Sources/ShopifyAcceleratedCheckouts/Wallets/ShopPay/ShopPayButton.swift

```diff
@@ -96,18 +96,16 @@ internal struct Internal_ShopPayButton: View {
 }
 
 @available(iOS 17.0, *)
-struct ShopPayButton_Previews: PreviewProvider {
-    static var previews: some View {
-        let mockCommonConfiguration = ShopifyAcceleratedCheckouts.Configuration(
-            storefrontDomain: "test-shop.myshopify.com",
-            storefrontAccessToken: "test-token"
-        )
+#Preview {
+    let mockCommonConfiguration = ShopifyAcceleratedCheckouts.Configuration(
+        storefrontDomain: "test-shop.myshopify.com",
+        storefrontAccessToken: "test-token"
+    )
 
-        ShopPayButton(
-            identifier: .variant(variantID: "gid://Shopify/ProductVariant/123", quantity: 1),
-            eventHandlers: EventHandlers()
-        )
-        .padding()
-        .environment(mockCommonConfiguration)
-    }
+    ShopPayButton(
+        identifier: .variant(variantID: "gid://Shopify/ProductVariant/123", quantity: 1),
+        eventHandlers: EventHandlers()
+    )
+    .padding()
+    .environment(mockCommonConfiguration)
```

Comment:
Again I don't think this should be included in the source code. Would it make more sense to have in the sample?